src/java.base/share/classes/java/lang/Module.java

```diff
@@ -111,6 +111,8 @@
     // the module descriptor
     private final ModuleDescriptor descriptor;
 
+    // is this module a native module
+    private volatile boolean enableNativeAccess = false;
```

Comment:
Can you drop "= false", it's not needed and I don't think we want a volatile-write here.
Also the comment may date from a previous iteration as there isn't any concept of a "native module".